components/scream/tests/coupled/homme_physics/CMakeLists.txt

```diff
@@ -78,3 +86,20 @@ CONFIGURE_FILE(${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/extensions/
                ${CMAKE_CURRENT_BINARY_DIR}/data COPYONLY)
 CONFIGURE_FILE(${SCREAM_BASE_DIR}/../eam/src/physics/rrtmgp/external/extensions/cloud_optics/rrtmgp-cloud-optics-coeffs-lw.nc
                ${CMAKE_CURRENT_BINARY_DIR}/data COPYONLY)
+
+## Finally compare all MPI rank output files against the single rank output as a baseline, using CPRNC
+## Only if max mpi ranks is >1
+# This test requires CPRNC
+include (BuildCprnc)
+BuildCprnc()
+SET (BASE_TEST_NAME "homme_shoc_cld_p3_rrtmgp")
+foreach (MPI_RANKS RANGE 2 ${SCREAM_TEST_MAX_RANKS})
```

Comment:
Careful! if SCREAM_TEST_MAX_RANKS is 1, this will enter undefined-behavior-land... You should either run starting from 1 (will run a cprnc with same src and tgt file) or wrap this chunk of code in an if statement.